package.json

```diff
@@ -25,13 +30,12 @@
     "rimraf": "^5.0.5"
   },
   "scripts": {
-    "prepack": "npm run clean && npm run compile-esm && npm run test && npm run remove-sourcemap-refs",
-    "compile": "tsc -p ./src && npm run lint",
-    "compile-esm": "tsc -p ./src/tsconfig.esm.json",
+    "prepack": "npm run clean && npm run test && npm run remove-sourcemap-refs",
+    "compile": "tsc -b && node ./build/fix-esm.mjs && npm run lint",
     "remove-sourcemap-refs": "node ./build/remove-sourcemap-refs.js",
```

Comment:
It seems that remove-sourcemap-refs job is useless. We can set `"sourceMap": false` in tsconfig. @aeschli

Reply:
That breaks code coverage and source mapping when running tests.